frontend/src/main/web/src/components/Notification.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import {
+  ButtonRound,
+  Icon,
+  Modal,
+  Row
+} from './'
+import {
+  clearMessage
+} from '../actions/common'
+import { isEmptyOrNull } from '../utils/StringUtils'
+
+class Notification extends Component {
```

Comment:
Needs doc. Looks like this is maybe a modal notification that overlays the page?